### PR TITLE
[zh] sync releases/1.18.x 1.17.x

### PR DESCRIPTION
**Pull request by my-git9**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/releases/1.18.x/announcing-1.18/_index.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+publishdate: 2023-06-07
+release: 1.18.00
+aliases:
+    - /zh/news/announcing-1.18
+    - /zh/news/announcing-1.18.0
+---
+
+我们很高兴地宣布 Istio 1.18 发布。这是 2023 年的第二个 Istio
+版本，也是第一个带有 Ambient 模式的版本！我们要感谢整个 Istio
+社区对 1.18.0 版本发布所作出的帮助。我们要感谢此版本的几位发布经理，
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18/_index.md:15]:
```suggestion
社区对 1.18.0 版本发布所作出的帮助。我们要感谢此版本的几位发布经理：
```

File: content/zh/news/releases/1.18.x/announcing-1.18/_index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+title: Announcing Istio 1.18.0
```
Comment (hanxiaop) [inline content/zh/news/releases/1.18.x/announcing-1.18/_index.md:2]: There are also change notes and upgrade notes included in the 1.18 release. I think they should be translated together.

File: content/zh/news/releases/1.18.x/announcing-1.18/_index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+---
+
+我们很高兴地宣布 Istio 1.18 发布。这是 2023 年的第二个 Istio
+版本，也是第一个带有 Ambient 模式的版本！我们要感谢整个 Istio
+社区对 1.18.0 版本发布所作出的帮助。我们要感谢此版本的几位发布经理，
+来自 Tetrate 的 `Paul Merrison`、来自 Microsoft 的 `Kalya Subramanian`
+和来自 DaoCloud 的 `Xiaopeng Han`。这些发布经理们要特别感谢测试和发布工作组负责人
+Eric Van Norman (IBM) 在整个发布周期中提供的帮助和指导。我们还要感谢 Istio
+工作组的维护者和更广泛的 Istio 社区，感谢他们在整个发布过程中通过及时的反馈、
+评论、社区测试以及您的所有支持来帮助我们确保及时发布。
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18/_index.md:20]:
```suggestion
工作组的维护者以及广大 Istio 社区，在发布过程中提供及时反馈、
审核和社区测试，以及在确保及时发布方面给予的全力支持。
```

File: content/zh/news/releases/1.18.x/announcing-1.18/_index.md (hunk 2 above)
Comment (hanxiaop) [reply content/zh/news/releases/1.18.x/announcing-1.18/_index.md:2]: https://github.com/istio/istio.io/blob/7ecfdf222e1aa762e772da1755eefe9d51b76d71/content/en/news/releases/1.18.x/announcing-1.18/change-notes/index.md

File: content/zh/news/releases/1.18.x/announcing-1.18/_index.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+{{< tip >}}
+Istio 1.18.0 已得到 Kubernetes `1.24` 到 `1.27` 的官方正式支持。
+{{< /tip >}}
+
+## 新特性 {#what-is-new}
+
+### Ambient Mesh
+
+Istio 1.18 标志着 Ambient Mesh 的首次发布，这是一种新的 Istio 数据平面模式，
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18/_index.md:32]:
```suggestion
Istio 1.18 标志着 Ambient Mesh 的首次发布，这是一种全新的 Istio 数据平面模式，
```

File: content/zh/news/releases/1.18.x/announcing-1.18/_index.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+旨在简化操作、更广泛的应用程序兼容性并降低基础设施成本。有关详细信息，
+请参阅公告[博客](/zh/blog/2022/introducing-ambient-mesh/)。
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18/_index.md:34]:
```suggestion
请参阅[官宣博文](/zh/blog/2022/introducing-ambient-mesh/)。
```

File: content/zh/news/releases/1.18.x/announcing-1.18/_index.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+### 网关 API 支持改进 {#gateway-api-support-improvements}
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18/_index.md:36]:
```suggestion
### Gateway API 支持改进 {#gateway-api-support-improvements}
```

File: content/zh/news/releases/1.18.x/announcing-1.18/_index.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+Istio 1.18 改进了对 Kubernetes Gateway API 的支持，
+包括对额外 v1beta1 资源的支持以及对自动化部署逻辑的增强，
+不再依赖于 Pod 注入。Istio 上的 Gateway API
+用户应查看此版本的升级说明以获取有关升级的重要指导。
+
+### 并发性设置的改变 {#proxy-concurrency-changes}
+
+以前，concurrency 设置在边车和网关的不同安装机制之间，存在不一致。
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18/_index.md:45]:
```suggestion
以前，代理 `concurrency` 设置（用于配置代理运行的工作线程数量）在 Sidecar 和不同的网关机制之间存在不一致的配置。
```

File: content/zh/news/releases/1.18.x/announcing-1.18/_index.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+有关此更改的更多详细信息，请参阅此版本的升级说明。
+
+### Istioctl 的增强 {#enhancements-to-the-istioctl-command}
+
+为 istioctl 命令添加了一些增强功能，包括对错误报告过程的增强功能和对
+istioctl analyze 命令的各种改进。
+
+## 升级至 1.18 {#upgrading-to-1.18}
+
+我们想听听您关于升级到 Istio 1.18 的体验。
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18/_index.md:56]:
```suggestion
我们期待倾听您关于升级到 Istio 1.18 的体验。
```

**Review by windsonsea — APPROVED** [review]
LGTM. Change and upgrade notes can be covered later since this PR already has a size/L.

**Comment by ericvn** [comment]
Appears a portion of the 1.18 code was sync'd to 1.18, but a portion was not.

**Pull request merged**